templates/education.html

```diff
@@ -0,0 +1,39 @@
+<div class="row">
+    <div class="col">
+        <h4>Education</h4>
+    </div>
+    <div class="col">
+        <ul class="pagination justify-content-end">
+            <li class="page-item">
+                <a id="education__previous" class="page-link" href="#" aria-label="Previous">
+                    <span aria-hidden="true">&laquo;</span>
+                </a>
+            </li>
+            <li class="page-item">
+                <a id="education__next" class="page-link" href="#" aria-label="Next">
+                    <span aria-hidden="true">&raquo;</span>
+                </a>
+            </li>
+        </ul>
+    </div>
+</div>
+<div class="row">
+    <div class="col">
+        <% for education in educations %>
+        <form class="education" id='<<"education__{:?}"|format(loop.index0)>>'>
```

Comment:
Maybe all the HTML files should use single quotes due to `'<<"education__{:?}"|format(loop.index0)>>'` for consistency?

Reply:
That would be more consistent, but also somewhat uncommon. IMO it's better as it is.